lib/view_model/send/output.dart

```diff
@@ -121,11 +121,19 @@ abstract class OutputBase with Store {
         return solana!.getEstimateFees(_wallet) ?? 0.0;
       }
 
-      final fee = _wallet.calculateEstimatedFee(
+      int? fee = _wallet.calculateEstimatedFee(
           _settingsStore.priority[_wallet.type]!, formattedCryptoAmount);
 
-      if (_wallet.type == WalletType.bitcoin ||
-          _wallet.type == WalletType.litecoin ||
+      if (_wallet.type == WalletType.bitcoin) {
+        if (_settingsStore.priority[_wallet.type]! == bitcoin!.getBitcoinTransactionPriorityCustom()) {
+          fee = _wallet.calculateEstimatedFeeWithFeeRate(
```

Comment:
here you can directly use `bitcoin!` since we already know that it's a bitcoin wallet.
thus you can remove `calculateEstimatedFeeWithFeeRate` from wallet base

Reply:
done